Makefile

```
@@ -169,3 +182,17 @@ amalthea_schema:  ## Updates generates pydantic classes from CRDs
 # newer than the requirements these steps won't be re-triggered.
 # Ignore the return value when there are more differences.
 	( git diff --exit-code -I "^#   timestamp\: " $@ >/dev/null && git checkout $@ ) || true
+
+# Devcontainer
+
+.PHONY: devcontainer_up
+devcontainer_up:
+	devcontainer up --workspace-folder .
+
+.PHONY: devcontainer_rebuild
+devcontainer_rebuild:
+	devcontainer up --remove-existing-container --workspace-folder .
+
+.PHONY: devcontainer_exec
+devcontainer_exec: devcontainer_up
+	devcontainer exec --container-id renku-data-services_devcontainer-data_service-1 -- bash
```

The default is the data services container. You dont have to set it.

```suggestion
	devcontainer exec -- bash
```